### ISSUE TITLE
Reveal results in multiple duplicate iotas

### ISSUE THREAD

**Issue by ZeekDaGeek** [SamsTheNerd/HexGloop#21]
For some unknown reason using Reveal when you have a pattern list causes each iota to be rendered multiple times.

![Bonus if you can spot the error I'm attempting to fix in this screenshot](https://github.com/SamsTheNerd/HexGloop/assets/1709288/524ed5e8-da7b-49a0-baa8-0f2103715927)

Just attempting to use `<northeast,qaq>` in chat only renders once.

I'm not sure how to copy a spell from an in game page to give you an example.

Minecraft version 1.9.2
Hex Gloop version 0.0.4 Forge
Hex Casting version 10.3
Modpack: All The Mods 8

**Edit:** maybe this will help? it's what shows up in the console for the chat message.
```
[<northwest,qwaeawqaeaqa> <southeast,aqaa> <east,ad> <west,qqq> <northeast,qaq> <east,aa> <east,qaq> <east,wa> <east,weaqa> <east,aadaa> <east,d> <east,ad> <west,qqq> <west,qqqaw>, "Please look at the entity you want to select first.", <northeast,de> <southwest,aqdee> <east,eee> <west,qqq> <west,qqqaw>, "Now select the Impetus you wish to deposit this mind into.", <northeast,de> <southwest,aqdee> <east,eee> <southeast,awdd> <southeast,deaqq> <east,eee> <west,qqq> <northwest,qwaeawqaeaqa> <southeast,aqaaw> <east,ad> <west,qqq> <northeast,qaq> <east,aa> <northeast,qaq> <east,wqaawdd> <southeast,dedqaqeeewewedqeqqqeqqqeqaqeq> <east,eee> <west,qqq> <west,qqqaw>, "Make sure your stack is empty and try again, selecting the Entity first, then the Impetus second.", <northeast,de> <southwest,aqdee> <east,eee> <southeast,awdd> <southeast,deaqq> <east,eee> <southeast,awdd> <southeast,deaqq>]
```

**Comment by SamsTheNerd** [comment]
yeah it's a known issue, i *think* i fixed it for the next update but i'll make sure to check through it again before release.

If you're curious why it's doing this, the pattern rendering is set up to store a pattern as a Style (same as text color and bold and italics and all that). Originally I just applied this style to a single character but that meant that it wasn't printing the pattern signature to the logs and some people wanted to keep that, so i switched to printing the whole thing and putting a boolean on the style to tell it not to render after the first one, guess something broke along the way and it decided to render it for each character instead !

**Comment by SamsTheNerd** [comment]
should be fixed in v0.1.0 - haven't had a chance to test on a server yet though, so feel free to reopen if there's still an issue.

**Comment by ZeekDaGeek** [comment]
The issue seems to still exist on servers.

**Comment by SamsTheNerd** [comment]
ok should be fixed for realzies this time with v0.1.1 !
